stablehlo/reference/tests/StablehloInterpreterRunner.cpp

```diff
@@ -0,0 +1,122 @@
+/* Copyright 2022 The StableHLO Authors.
```

Comment:
Let's move this file to `stablehlo/tools`, next to `stablehlo-opt`, and rename to `StablehloInterpreterMain.cpp` for consistency?